README.md

```diff
@@ -1,2 +1,140 @@
+
+```sh
+libp2p-relay-server [--peerId <jsonFilePath>] [--listenMultiaddrs <ma> ... <ma>] [--announceMultiaddrs <ma> ... <ma>] [--metricsMultiaddr <ma>] [--disableMetrics] [--disablePubsubDiscovery] [--discoveryTopics <topic> ... <topic>]
+```
+
+#### PeerId
+
+You can create a [PeerId](https://github.com/libp2p/js-peer-id) via its [CLI](https://github.com/libp2p/js-peer-id#cli). 
+
+```sh
+libp2p-relay-server --peerId id.json
+```
+
+#### Multiaddrs
+
+You can specify the libp2p rendezvous server listen and announce multiaddrs. This server is configured with [libp2p-tcp](https://github.com/libp2p/js-libp2p-tcp) and [libp2p-websockets](https://github.com/libp2p/js-libp2p-websockets) and addresses with this transports should be used. It can always be modified via the API.
+
+```sh
+libp2p-relay-server --peerId id.json --listenMultiaddrs '/ip4/127.0.0.1/tcp/15002/ws' '/ip4/127.0.0.1/tcp/8000' --announceMultiaddrs '/dns4/test.io/tcp/443/wss/p2p/12D3KooWAuEpJKhCAfNcHycKcZCv9Qy69utLAJ3MobjKpsoKbrGA' '/dns6/test.io/tcp/443/wss/p2p/12D3KooWAuEpJKhCAfNcHycKcZCv9Qy69utLAJ3MobjKpsoKbrGA'
+```
+
+By default it listens on `/ip4/127.0.0.1/tcp/15002/ws` and has no announce multiaddrs specified.
+
+#### Metrics
+
+Metrics are enabled by default on `/ip4/127.0.0.1/tcp/8003` via Prometheus. This address can also be modified with:
+
+```sh
+libp2p-relay-server --metricsMultiaddr '/ip4/127.0.0.1/tcp/8000'
+```
+
+Moreover, metrics can also be disabled with:
+
+```sh
+libp2p-relay-server --disableMetrics
+```
+
+#### Discovery
+
+A discovery module [libp2p/js-libp2p-pubsub-peer-discovery](https://github.com/libp2p/js-libp2p-pubsub-peer-discovery) is configured and enabled by default. It can be disabled with:
+
+```sh
+libp2p-relay-server --disablePubsubDiscovery
+```
+
+This discovery module uses `_peer-discovery._p2p._pubsub` as a topic for enabling peer discovery. However, other topics can be used with:
+
+```sh
+libp2p-relay-server --discoveryTopics '_peer-discovery._app1._pubsub' '_peer_discovery._app2._pubsub'
+```
+
+### Docker
+
+When running the relay server in Docker, you can configure the same parameters via environment variables, as follows:
+
+```sh
+PEER_ID='./id.json'
+LISTEN_MULTIADDRS='/ip4/127.0.0.1/tcp/15002/ws,/ip4/127.0.0.1/tcp/8000'
+ANNOUNCE_MULTIADDRS='/dns4/test.io/tcp/443/wss/p2p/12D3KooWAuEpJKhCAfNcHycKcZCv9Qy69utLAJ3MobjKpsoKbrGA,/dns6/test.io/tcp/443/wss/p2p/12D3KooWAuEpJKhCAfNcHycKcZCv9Qy69utLAJ3MobjKpsoKbrGA'
+METRICS_MULTIADDR='/ip4/127.0.0.1/tcp/8000'
+DISABLE_METRICS='true'
+DISABLE_PUBSUB_DISCOVERY='true'
+DISCOVERY_TOPICS='_peer-discovery._app1._pubsub,_peer_discovery._app2._pubsub'
+```
+
+Please note that you should expose expose the used ports with the docker run command. The default ports used are `8003` for the metrics and `150003` for the websockets listener
```

Comment:
```suggestion
Please note that you should expose the used ports with the docker run command. The default ports used are `8003` for the metrics, `8000` for the TCP transport, and `15002` for the Websockets transport
```

Reply:
Also, I don't see any defaults aside from metrics. The docker commands include them, but there aren't defaults in the code unless I am missing something.

Reply:
Ok, perhaps the example below created confusion. I added now `Configuration example` before it.

So, the default is in the `getListenAddresses` and is '/ip4/127.0.0.1/tcp/15003/ws'. It is also mentioned above in the readme on the multiaddrs subsection. Should I also add a default tcp multiaddr?